cmake/MacroASN1Header.cmake

```diff
@@ -34,5 +34,9 @@ macro(add_asn1_headers _groupname)
 		add_asn1_header(${_header})
 		add_dependencies(${_groupname} ${_header}.h)
 	endforeach()
+	add_custom_target (symlink-include-${_groupname}
+		ln -s ${CMAKE_CURRENT_BINARY_DIR}  ${CMAKE_CURRENT_BINARY_DIR}/quick-der
```

Comment:
Hmm, we moeten eens kijken of we dit wat meer portable kunnen maken, want `ln -s` werkt niet op Windoze.
Misschien gewoon een subdir `quick-der` maken en daar de output in plaatsen.